### DISCUSSION TITLE
When will a new version be deployed onto npm?

### DISCUSSION THREAD

**Discussion by morgan-bp** [silevis/reactgrid#207] [General]
There are some features on the develop branch that I would like to use ([notably the moveRightOnEnterProp](https://github.com/silevis/reactgrid/commit/56a96c380434fc9c8863f9c737d85e3172a505c6)).

When will a new version with new features be up on npm?

**Comment by Michota** [comment]
I am 95% sure that it will be available by the end of the first week of 2024.

**Reply by DLowHP** [reply]
Probably. But the mentioned `moveRightOnEnter` is available in the (already released) version [v4.1.0](https://github.com/silevis/reactgrid/releases/tag/v4.1.0) and above. @morgan-bp
